concent_api/core/models.py

```diff
@@ -485,3 +493,478 @@ def clean(self):
             raise ValidationError({
                 'pending_response': 'PaymentInfo should be related with Pending Response'
             })
+
+
+class AbstractMessageModel(Model):
+
+    sig = BinaryField()
+    timestamp = IntegerField()
+    encrypted = BooleanField(default = False)
+
+    class Meta:
+        abstract = True
+
+
+class StoredComputeTaskDef(AbstractMessageModel):
+
+    # Defines items list from ComputeTaskDef
+    ITEMS = [
+        'task_id',
+        'subtask_id',
+        'deadline',
```

Comment:
You're not checking in any way that this list contains the names of fields defined in the model. Someone can just come here, ignore your list, add a new field to the model below and the checks will still pass.

Instead of defining `ITEMS` here you should be comparing slots in golem-messages with the list of model fields obtained from [get_fields()](https://docs.djangoproject.com/en/2.0/ref/models/meta/#django.db.models.options.Options.get_fields).